src/tsickle.ts

```diff
@@ -1209,6 +1223,46 @@ class Annotator extends ClosureRewriter {
     return emitText;
   }
 
+  /** Emits a variable statement per declaration in this variable statement, duplicating JSDoc. */
+  private visitVariableStatement(varStmt: ts.VariableStatement, docTags: jsdoc.Tag[]) {
+    // Previously tsickle would emit inline types (`var /** @type {string} */ x;`), but TypeScript's
+    // emit strips those comments for exports.
```

Comment:
Can you complete this comment with what we do now?  E.g.
// So instead we now emit [...].

Reply:
Done.